Antidote/AbstractCallViewController.h

```diff
@@ -8,31 +8,45 @@
 
 #import <UIKit/UIKit.h>
 
+#import "OCTToxAVConstants.h"
+
 @class OCTSubmanagerCalls;
 @class OCTChat;
 @class OCTCall;
 
+@protocol AbstractCallViewControllerDelegate <NSObject>
+
+@required
+- (void)dismissCurrentCall;
+- (void)callAccept;
+- (BOOL)sendCallControl:(OCTToxAVCallControl)control error:(NSError **)error;
+
+- (void)otherCallAccept:(BOOL)accept;
+
+@property (nonatomic, assign) BOOL enableMicrophone;
+
+@end
```

Comment:
I have some ideas about improving this protocol. Don't have time now, will write you in a 2-3 hours. :-)